sensors:
  break_beam_1:
    type: break_beam
    pose:
      xyz: [1.6, 1.389, 0.91]
      rpy: [0, 0, 'pi']
  proximity_sensor_1:
    type: proximity_sensor
    pose:
      xyz: [0.95, 2.6, 0.92]
      rpy: [0, 0, 0]
  logical_camera_1:
    type: logical_camera
    pose:
      xyz: [1.279, 1.97, 1.429]
      rpy: [2.645, 1.5, 2.63]
  logical_camera_2:
    type: logical_camera
    pose:
      xyz: [-0.687, 0.378, 0.919]
      rpy: [0.042, 0.682, 0.011]
  logical_camera_3:
    type: logical_camera
    pose:
      xyz: [-0.624, -0.438, 0.895]
      rpy: [0.042, 0.682, 0.011]
  logical_camera_4:
    type: logical_camera
    pose:
      xyz: [0.3, 3.15, 1.5]
      rpy: [0, 'pi/2', 0]
  logical_camera_5:
    type: logical_camera
    pose:
      xyz: [0.3, -3.15, 1.5]
      rpy: [0, 'pi/2', 0]
  depth_camera_1:
    type: depth_camera
    pose:
      xyz: [-0.3, -1.148, 1.2]
      rpy: [0, 1.5707, 0]
  laser_profiler_1:
    type: laser_profiler
    pose:
      xyz: [1.21, 4, 1.6]
      rpy: ['-pi', 'pi/2', 'pi/2']
